vendor/github.com/Microsoft/opengcs/service/gcsutils/remotefs/remotefs.go

```diff
@@ -287,90 +288,115 @@ func Mkfifo(in io.Reader, out io.Writer, args []string) error {
 //  - args[1] = flag in base 10
 //  - args[2] = permission mode in octal (like 0755)
 func OpenFile(in io.Reader, out io.Writer, args []string) (err error) {
+	logrus.Debugf("OpenFile: %v", args)
```

Comment:
Logging has become really noisy in this file; hope this is only temporary, and to be removed at some point; errors returned are likely logged again by dockerd, so may be logged twice